apps/omg/lib/omg/ethereum_event_listener.ex

```diff
@@ -188,12 +188,13 @@ defmodule OMG.EthereumEventListener do
     :timer.send_after(ethereum_events_check_interval_ms, self(), :sync)
   end
 
-  defp publish_data([%{event_signature: event_signature} | _] = data) do
-    # event signature is string with a method name with arguments,
-    # for example: BlockSubmitted(uint256)
+  defp publish_events([]), do: :ok
+
+  defp publish_events([%{event_signature: event_signature} | _] = data) do
     [event_signature, _] = String.split(event_signature, "(")
-    OMG.Bus.direct_local_broadcast(event_signature, {:data, data})
-  end
 
-  defp publish_data([]), do: :ok
+    event_signature
```

Comment:
this was an unnecessary change.

Reply:
It just happened. I was moving it back and forth between modules. If you prefer having `  defp publish_data([]), do: :ok` I can move it up for you :+)